installer/templates/phx_test/controllers/error_html_test.exs

```diff
@@ -2,7 +2,7 @@ defmodule <%= @web_namespace %>.ErrorHTMLTest do
   use <%= @web_namespace %>.ConnCase, async: true
 
   # Bring render_to_string/4 for testing custom views
-  import Phoenix.Template
+  import Phoenix.Template, only(render_to_string: 4)
```

Comment:
I believe what we want is:

```suggestion
  import Phoenix.Template, only: [render_to_string: 4]
```

Reply:
Strange, I had that before but the formatter check fails:
```
  1) test phx.gen.auth + bcrypt has no compilation or formatter warnings (--live) (Phoenix.Integration.CodeGeneration.UmbrellaAppWithDefaultsTest)
     test/code_generation/umbrella_app_with_defaults_test.exs:180
     ** (RuntimeError) mix command failed with exit code: 1

     mix format --check-formatted

     ** (Mix) mix format failed due to --check-formatted.
     The following files are not formatted:

     /phoenix/installer/tmp/qvroDhgA8i/new with defaults/rainy_day_umbrella/apps/rainy_day_web/test/rainy_day_web/controllers/error_html_test.exs
     
            |
      3  3  |
      4  4  |  # Bring render_to_string/4 for testing custom views
      5    -|  import Phoenix.Template, only [render_to_string: 4]
         5 +|  import Phoenix.Template, only(render_to_string: 4)
      6  6  |
      7  7  |  test "renders 404.html" do
            |
```

Reply:
ah missing the extra colon, let me try that

Reply:
that worked, thank you @Gazler